tests/dataset_runner.py

```diff
@@ -219,7 +220,7 @@ def _assert_data_browser_bundles(self, project_shortname):
             response_json = self.azul_agent.get_specimen_by_project(project_shortname)
             orange_uuids = {bundle["bundleUuid"] for hit in response_json["hits"] for bundle in hit["bundles"]}
             assert orange_uuids == set(self.primary_bundle_uuids).union(self.secondary_bundle_uuids)
-            orange_shortnames = {project["projectShortname"] for hit in response_json["hits"] for project in hit["projects"]}
+            orange_shortnames = {project["projectShortname"][0] for hit in response_json["hits"] for project in hit["projects"]}
```

Comment:
Generally speaking, you should consider every element in the list, not just the first. Alternatively, you should assert that the list is a singleton (is of length 1). I think given the nature of the spreadsheet, the latter would be more appropriate. It creates a new specimen entity for every test run so it should be impossible for a specimen created by the test to refer to more than one project.